documentation/developers/rfid/generic_nfcpy.md

```diff
@@ -0,0 +1,24 @@
+# Generic NFCPy Reader
+
+This module is based on the user space NFC reader library [nfcpy](https://nfcpy.readthedocs.io/en/latest/overview.html) ([on github](https://github.com/nfcpy/nfcpy)).
+The link above also contains a list of supported devices.
+
+The goal of this module is to handle USB NFC devices, that don't have a HID-keyboard
+driver, and thus cannot be used with the [genericusb](genericusb.md) module. 
+
+## Installation
+
+Since nfcpy is a user-space library, it is required to supress the kernel from loading its driver:
```

Comment:
I suggest to create a setup.inc.sh script, which performs all these steps. This script (along with the requirements.txt) is picked up by the RFID configuration tool and so also your addition is integrated in the regular installation process.

see https://github.com/MiczFlor/RPi-Jukebox-RFID/tree/future3/develop/src/jukebox/components/rfid/hardware/rc522_spi for an example.